src/ui/qgsstyleitemslistwidgetbase.ui

```diff
@@ -251,7 +251,14 @@
         <number>0</number>
        </property>
        <item>
-        <widget class="QTreeView" name="mSymbolTreeView"/>
+        <widget class="QTreeView" name="mSymbolTreeView">
+         <property name="minimumSize">
+          <size>
+           <width>0</width>
+           <height>120</height>
```

Comment:
That's not really hi-dpi friendly -- you'd need to set it in c++, using a multiple of the font metrics height...

Reply:
Really? I thought these unit-less values in .ui for hi-dpi friendly. Darn, TIL :cry: